pkg/simulation/target_test.go

```diff
@@ -23,7 +23,54 @@ func NewSim(t *testing.T, seed int64) (*simulation.Simulation, *mock.MockAttribu
 	return sim, attr
 }
 
-func TestRetargetSimple(t *testing.T) {
+// test var : Targets list
+func TestRetargetEmptyTargets(t *testing.T) {
+	sim, attr := NewSim(t, 1)
+
+	// for all targets, return full health in this test
+	attr.EXPECT().HPRatio(gomock.Any()).Return(1.0).AnyTimes()
```

Comment:
Remove this `EXPECT`. For this test, we shouldn't expect `attr` to be called (since no input targets). By removing the `EXPECT`, the test will fail if for some reason we suddenly call `attr.HPRatio`